src/version.h

```diff
@@ -1,7 +1,8 @@
 /* Version examples:
 	"0.8.0"			release
 	"0.8.0.dev1"	development, see https://www.python.org/dev/peps/pep-0440
+    "0.8.0+dcl3"    local branch patches, before upstreaming
 
 	 NB: OPENQL_VERSION_STRING is also decoded by setup.py
 */
-#define OPENQL_VERSION_STRING "0.8.1.dcl3"
+#define OPENQL_VERSION_STRING "0.8.1+dcl3"
```

Comment:
This is still an illegal name. A legal name can be only 0.8.1.dev\<NUMBER\>. Adding other characters such as "+" or "dc" is not permitted. The only string allowed is "dev", so that is not an issue, i.e., no need to change the "." to "+"

Reply:
thanks for your comment, I was looking for a way to give a meaningful version name to an upstream branch. This "+" syntax seemed to be allowed by the PEP, but I'll change to "dev" since it doesn't work